packages/docs/docs/getting-started/media.mdx

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_position: 13
+slug: /media
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Media
+
+In this chapter we will take a look at all the different types of media Motion
+Canvas lets you add to your animations.
+
+## Audio
+
+One type of media Motion Canvas supports is audio. To add an audio track to your
+animation you would need to edit your `project.ts` file. First we need to import
+the audio file we want to use as follows:
+
+```ts
+// my-animation/
+// ├─ audio/
+// │  └─ voice.mp3
+// ├─ src/
+// ...
+import audio from '../audio/voice.mp3';
+```
+
+One imported we now need to let Motion Canvas know that we want to use this
+audio file. We can do this by specifying the
+[`audio`](/api/core/index/ProjectConfig#audio) property in the
+[`ProjectConfiguration`](/api/core/index/ProjectConfig):
+
+```ts
+export default makeProject({
+  // ...
+  audio: audio,
+});
+```
+
+If we open the editor now we should see that your audio track has been added.
+
+![](@site/static/img/media/editor-audio-track.png)
+
+However sometimes its not ideal to start with the audio at the beginning but
+rather with a bit of a delay. This can be done by using the
+[`audioOffset`](/api/core/index/ProjectConfig#audioOffset) property and
+specifying a amount of seconds.
+
+```ts
+export default makeProject({
+  // ...
+  audio: audio,
+  audioOffset: 1.5, // delays the audio by 1.5 second
+});
+```
+
+![](@site/static/img/media/editor-audio-track-delayed.png)
+
+### Supported Formats
+
+A list of supported file formats can be found
+[here](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Audio_codecs#common_codecs)
+
+## Images
+
+Another type of media Motion Canvas supports are images. To use an image in the
+animation we first need to import the image:
+
+```ts
+// my-animation/
+// ├─ images/
+// │  └─ example.png
+// ├─ src/
+// ...
+import examplePng from '../../images/example.png';
+```
+
+Then we can use Motion Canvas [`Image`](/api/2d/components/Image) component to
```

Comment:
nit: "as follows" and colon tend to have the same purpose. I'd omit "as follows" to keep it more straightforward.